specs/validator/0_beacon-chain-validator.md

```diff
@@ -329,19 +329,19 @@ Next, the validator creates `attestation`, an [`Attestation`](../core/0_beacon-c
 
 Set `attestation.data = attestation_data` where `attestation_data` is the `AttestationData` object defined in the previous section, [attestation data](#attestation-data).
 
-##### Aggregation bitfield
+##### Aggregation bits
 
-* Let `aggregation_bitfield` be a byte array filled with zeros of length `(len(committee) + 7) // 8`.
+* Let `aggregation_bits` be a byte array filled with zeros of length `(len(committee) + 7) // 8`.
```

Comment:
Suggest we use a `Bitlist` here and use the `0b1` notation and simple array index for setting the bit